src/Config/Config.ts

```diff
@@ -16,12 +16,14 @@ export default class Config implements IConfigParams {
   provider: any;
   scanSpread: any;
   statsDb: StatsDB;
+  util: any;
```

Comment:
Change the type from `any` to `Util`.

Reply:
Is it then necessary for the cli to also forward the Util object to the client?
If it is, that is not a very elegant solution. Couldn't we use the Util directly without requiring it as a parameter to Config?

Reply:
I also noticed a circular dependency - the Config takes a Util as a parameter, but the Util also has a Config setter in its constructor.

Reply:
The Util needs the web3 object, and it seems most logical just to peg it onto the config, since it will be used in various parts of the TimeNode.

Reply:
Ok I see that you made those exact changes now :)